airqo-platform/AirQo-api
# Fix failing deployments

**_WHAT DOES THIS PR DO?_**
- [x] [add missing dependecy: image-tag](https://github.com/airqo-platform/AirQo-api/pull/1216/commits/4e498a997b58f829d622b623e6b8500f3c2994bf)
- [x] [correct kafka-connector path](https://github.com/airqo-platform/AirQo-api/pull/1216/commits/b8eee9e901fb8fe8d16290115e752bf18b1bd176)
- [x] [correct meta-data registry variables](https://github.com/airqo-platform/AirQo-api/pull/1216/commits/8179126c062bdcc939c92dd1da108b4b57a77e83)
- [x] [reference right channel data image](https://github.com/airqo-platform/AirQo-api/pull/1216/commits/52e567ec81310232a91a6237a786a951d8ef9042)
- [x] [update the build push action version](https://github.com/airqo-platform/AirQo-api/pull/1216/commits/422ecde7829e79c9be18d84d0695a786aec6e00b)
- [x] [remove deprecated use of 'updateFile'](https://github.com/airqo-platform/AirQo-api/pull/1216/commits/8be3429e09829236c151d2881baef930f14b0220)
- [x] [trigger all workflows to run](https://github.com/airqo-platform/AirQo-api/pull/1216/commits/1783dca81c59c59d5fd004467a512085794fc5f5)
- [x] [update docker login action version](https://github.com/airqo-platform/AirQo-api/pull/1216/commits/055eab7de17ed072a99813b0ce1354f84ec14d59)
- [x] [update k8s-context action version](https://github.com/airqo-platform/AirQo-api/pull/1216/commits/cf72bd177d9aa68dc4f741cd700df1be326ff1c7)
- [x] [update files with retry](https://github.com/airqo-platform/AirQo-api/pull/1216/commits/9ded1a6e12446b99e612be513e35166b9edcb62f)

**_WHAT ISSUES ARE RELATED TO THIS PR?_**
- Closes #1215 

**_HOW DO I TEST OUT THIS PR?_**
- Checkout the test run workflow [here](https://github.com/airqo-platform/AirQo-api/actions/runs/3638348762)
- Note that the Kafka-connectors job is still in development cc @NoahNsimbe 
- ![image](https://user-images.githubusercontent.com/41787587/206167835-b45abe7c-14a2-45ea-93d1-dc4079905fe9.png)

**_WHICH ENDPOINTS SHOULD BE READY FOR TESTING?:_**
- n/a

**_IS THERE ANY JENKINS CONSOLE LINK FOR CODE COVERAGE AND BUILD INFO?_**
- n/a

**_ARE THERE ANY RELATED PRs?_**
- n/a
